addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -108,6 +124,53 @@ func _drop_data(at_position: Vector2, data: Variant) -> void:
 	reconnect_block.emit(block)
 
 
+func _drop_obj_property(at_position: Vector2, data: Variant) -> void:
+	var obj_name = str(data.get("object", "")).get_slice(":", 0)
+	var prop_name = data.get("property", "")
+	var val = data.get("value", null)
```

Comment:
Same nits a above for the `val` variable here. It is not a great variable name, and I'm not sure if the extra safety is needed.